primo-zip.py

```diff
@@ -41,19 +43,16 @@ def zip_up_source(zip_name, base_directory, directory):
 
 
 def main():
+    # user must input zip_name parameter at command line
+    if len(sys.argv) > 1:
+        zip_name = sys.argv[1]
+    else:
+        sys.exit("Folder name parameter must be specified.")
+
     # zip info
     types_to_check = ('.js', '.css', 'html', 'md', 'txt')
     base_directory = '.'
-    directory = './01UCS_LAL-UCLA'
-    directory_test = './01UCS_LAL-Test_00'
-    zip_name = '01UCS_LAL-UCLA'
-    zip_name_test = '01UCS_LAL-Test_00'
-
-    # use the test folder if it is present
-    isdir = os.path.isdir(directory_test)
-    if(isdir):
-        directory = directory_test
-        zip_name = zip_name_test
+    directory = './' + zip_name
```

Comment:
@ztucker4 Please add a check for the existence of the `zip_name` directory.  Otherwise, the program both crashes and creates an (empty) zip file for the non-existent directory.
```
$ python primo-zip.py asd
... crash, caused by FileNotFoundError: [Errno 2] No such file or directory: 'asd'

File asd.zip is created
$ unzip -l asd.zip
Archive:  asd.zip
warning [asd.zip]:  zipfile is empty
```
It's OK not to catch this error as running the program is meaningless without a real directory, but the program should exit right after this line if the `zip_name` directory does not exist.